server/src/main/java/org/opensearch/script/ScriptService.java

```diff
@@ -106,8 +106,7 @@
     public static final Setting.AffixSetting<TimeValue> SCRIPT_CACHE_EXPIRE_SETTING =
         Setting.affixKeySetting(CONTEXT_PREFIX,
             "cache_expire",
-            key -> Setting.positiveTimeSetting(key, SCRIPT_GENERAL_CACHE_EXPIRE_SETTING, TimeValue.timeValueMillis(0),
```

Comment:
This is positive, so >= 0 is redundant. Note that this is probably a confused declaration as this setting has a fallback.